agent/agentssh/agentssh.go

```diff
@@ -125,6 +135,9 @@ func NewServer(ctx context.Context, logger slog.Logger, maxTimeout time.Duration
 			"streamlocal-forward@openssh.com":        unixForwardHandler.HandleSSHRequest,
 			"cancel-streamlocal-forward@openssh.com": unixForwardHandler.HandleSSHRequest,
 		},
+		X11Callback: func(ctx ssh.Context, x11 ssh.X11) bool {
+			return x11Callback(logger, fs, ctx, x11)
```

Comment:
```suggestion
			return x11Callback(s.logger, fs, ctx, x11)
```
Makes this function a bit more portable (perhaps fs could be stored on server as well).